.github/CODEOWNERS

```diff
@@ -1 +1 @@
-* @Esadruhn @Fabien-GELUS @maikia @louishulot
+* @Esadruhn @Fabien-GELUS @louishulot @RomainGoussault @ThibaultFy @oleobal @Milouu @guilhem-barthes
```

Comment:
```suggestion
* @Esadruhn @Fabien-GELUS @louishulot @ThibaultFy @oleobal @Milouu @guilhem-barthes
```
according to our contributing guide (on the private repo), code owners are:

> Person who knows well the code base and who is able to judge whether this can be merged in main or not. The code owners have the following responsibilities:
> 
> review PR before it can be merged into the main branch.
> resolve security vulnerabilities raised by our tooling (Dependabot, ...).
> handle Github Issues lifecycle.
> 
> Code owners are added by tech leads for each repo, everybody can suggest new code owners.

Only people who work on the codebase should be code owners